README.md

```diff
@@ -1,132 +1,135 @@
+may choose to omit glfw. Similarly, you can (of course) swap it for any other
+wgpu-compatible canvas, e.g., PyQt, PySide, or wx.
 
-Further we aim for a few niceties:
-* Proper support for high-res screens.
-* Builtin anti-aliasing.
-* Custom post-processing steps.
-* Support for picking objects and parts within objects.
-* (approximate) order-independent transparency (OIT) (not implemented yet).
+## Example
 
+> **Note**
+> A walkthrough of this example can be found in [the
+> guide](https://pygfx.readthedocs.io/en/latest/guide.html#how-to-use-pygfx).
 
-## WGPU is awesome (but also very new)
+```python
+import pygfx as gfx
 
-Working with the WGPU API feels so much nicer than OpenGL. It's well
-defined, no global state, we can use compute shaders, use storage
-buffers (random access), etc.
+cube = gfx.Mesh(
+    gfx.box_geometry(200, 200, 200),
+    gfx.MeshPhongMaterial(color="#336699"),
+)
 
-Fair enough, the WGPU API is very new and is still changing a lot, but
-eventually it will become stable. One of the biggest downsides right
-now is the lack of software rendering. No luck trying to run wgpu on a
-VM or CI.
+def animate():
+    rot = gfx.linalg.Quaternion().set_from_euler(
+            gfx.linalg.Euler(0, 0.01)
+        )
+    cube.rotation.multiply(rot)
 
-Because of how Vulkan et. al. work, the WGPU API is aimed at predefining
-objects and pipelines and then executing these. Almost everything is
-"prepared". The main reasoning for this is consistency and stable drivers,
-but it also has a big advantage for us Pythoneers: the amount of code per-draw-per-object
-is very limited. This means we can have *a lot* of objects and still be fast.
+if __name__ == "__main__":
+    gfx.show(cube, before_render=animate)
 
-As an example, see `collections_line.py`: drawing 1000 line objects with 30k points each at 57 FPS (on my laptop).
+```
+<img src="./docs/_static/guide_rotating_cube.gif" alt="drawing" width="400"/>
 
 
-## How to build a visialization
+## Features
+Some of pygfx's key features are:
 
-See also the examples, they all do something like this:
+- Native support for High Resolution screens.
+- Builtin anti-aliasing.
+- Support for picking objects and parts within objects.
 
-* Instantiate a renderer and a canvas to render to.
-* Create a scene and populate it with world objects.
-* Create a camera (and maybe a control).
-* Define an  animate function that calls: `renderer.render(scene, camera)`
+@almarklein: A few suggestions and feature discussions
+- fast draws (57 FPS @ 30M vertices from 1k lines on a . . .) (@almarklein: what
+  specs does your laptop have? Also, do we have a polygon count benchmark?)
+- (approximate) order-independent transparency (OIT). (is this implemented now?)
+- Custom post-processing. (do the examples for this need updating?)
+- Easy shader templating
+- built atop [WGPU](https://github.com/pygfx/wgpu-py) (>= OpenGL)
+- semantically similar to TreeJS (WorldObjects, decoupled rendering backend, ...)
```

Comment:
I think the event system is also worth a mention

Reply:
Idea: perhaps we can split out features particular of interest for 1) object rendering / meshes; 2) scientific viz.

Reply:
I would only list the 3 or 5 features that we are most proud of (or that sell the best) otherwise it quickly becomes a boring cloth-line.

For all the others, we should already have a feature example that shows off each feature right? So we can just point users to the gallery and say "check if the feature you want is listed here".

Reply:
I've made a selection of the top 5 features from all our suggestions. I kicked out some (light/shaddows, PBR, ...) that you could already see in the example images at the top, or that is mentioned elsewhere (TreeJS/WGPU) and then choose the 5 items that sounded the most impressive.

Suggestions are, ofc, welcome, and we also have to find the corresponding examples for each feature.